### PR TITLE
Add ruby_tuesday_us spider (216 locations)

### PR DESCRIPTION
**Pull request by sterojos**

### CODE REVIEW

**Comment by sterojos** [comment]
Fixes #6145

File: locations/spiders/ruby_tuesday_us.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from scrapy import Spider
+from scrapy.http import Response
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class RubyTuesdayUsSpider(Spider):
+    name = "ruby_tuesday_us"
+    item_attributes = {"brand": "Ruby Tuesday", "brand_wikidata": "Q7376400", "nsi_id": "rubytuesday-2fabb0"}
```
Comment (davidhicks) [inline locations/spiders/ruby_tuesday_us.py:13]: Are `nsi_id` values stable? We have `pipelines/apply_nsi_categories.py` which dynamically finds the relevant NSI entry by matching `brand_wikidata` and `operator_wikidata`, ensuring the latest `nsi_id` is used

File: locations/spiders/ruby_tuesday_us.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    start_urls = ["https://www.rubytuesday.com/locations/"]
+
+    def parse(self, response: Response):
+        scripts = response.css('[type="text/javascript"]::text').getall()
+        locations = parse_js_object([s for s in scripts if s.startswith("var locations_meta")][0])
```
Comment (davidhicks) [inline locations/spiders/ruby_tuesday_us.py:18]: Perhaps replace with something like:
```
js_blob = response.xpath('//script[contains(text(), "var locations_meta =")]/text()').get()
js_blob = js_blob.split(...  # etc, if necessary
for location in parse_js_object(js_blob):
```

File: locations/spiders/ruby_tuesday_us.py (hunk 1 above)
Comment (sterojos) [reply locations/spiders/ruby_tuesday_us.py]: I think it also blocked the automatic category detection. Thanks for the correction!

**Review by Cj-Malone — APPROVED** [review]
Thank you. LGTM.

https://alltheplaces-data.openaddresses.io/map.html?show=https://alltheplaces-data.openaddresses.io/ci/7702497110/ruby_tuesday_us/output.geojson

File: locations/spiders/ruby_tuesday_us.py (hunk 1 above)
Comment (Cj-Malone) [reply locations/spiders/ruby_tuesday_us.py]: `nsi_id` isn't stable, and yeah, that will have blocked the automatic categories. Thank you.

**Pull request merged**